01-python-basics/hw-01-python-basics.py

```diff
@@ -0,0 +1,51 @@
+# import lib with needed functions
+import random
+
+# create list of 100 random numbers from 0 to 1000
+# using sample method that returns a list of 100 elements in range between 0 and 1000
+random_list = random.sample(range(0, 1000), 100)
```

Comment:
It's new way for me (and it looks good) for creating such list, thanks
JFYI Another option is list comprehension:
random_list = [random.randint(0, 1000) for _ in range(100)]